src/index.js

```diff
@@ -326,6 +327,67 @@ function crossEnv(args) {
   return `${getBin('cross-env')} ${args}`
 }
 
+/**
+ * The options to pass to includePackage
+ * @typedef {Object|string} IncludePackageOptions
+ * @property {string} path - the path to the package scripts
+ */
+
+/**
+ * Includes the scripts from a sub-package in your repo (for
+ * yarn workspaces or lerna style projects).
+ * @param {IncludePackageOptions} packageNameOrOptions - either a
+ * simple name for the sub-package or the relative path to it.
+ *  If you just provide the name and not the path, then the path
+ *  defaults to: /packages/{package}/package-scripts.js
+ * @return {any} will return an object of scripts loaded from that package
+ */
+function includePackage(packageNameOrOptions) {
+  const packageScriptsPath = typeof packageNameOrOptions === 'string' ?
+    `./packages/${packageNameOrOptions}/package-scripts.js` :
+    packageNameOrOptions.path
+  
+  const startingDir = process.cwd()
+  const relativeDir = path.relative(startingDir,
+    path.dirname(packageScriptsPath)).split('\\').join('/')
```

Comment:
Will this work the same on Windows and Linux?

Reply:
Ah you know what, I think thats why I was having travis issues. Ill look at it now.